### ISSUE TITLE
Connection check problem

### ISSUE THREAD

**Issue by MarkoPuk01** [cubewise-code/tm1py#630]
Hi, 
I tried to check connection with **check.py** script, but I am constantly getting **'NoneType' object has no attribute 'text'**
In step before I successfully got $metadata, so I am pretty sure that there is some problem with connection.

I installed most recent TM1py (1.8) and TM1 Server is on version 11.5.
I tried first tried Python 3.10.0 and then transferred to Python 3.9.7., but that didn't solve problem.

**Configuration that I am filling on input**:

TM1 User (leave empty if SSO): user
Password (leave empty if SSO):pass
CAM Namespace (leave empty if no CAM Security):
Address (leave empty if localhost): https://dw-tm1paw207.pi.local
ClientCAMURI (leave empty if no SSO):
HTTP Port (Default 5000): 1234
SSL (Default T or F): T

**My config file**:

[TM1S]
DistributedPlanningOutputDir=tunitS
PersistentFeeders=T
ForceReevaluationOfFeedersForFedCellsOnDataChange=T
DataBaseDirectory=DataFiles
LoggingDirectory=LogFiles
ServerName=Server_Name
PortNumber=123456
HTTPPortNumber=1234
UseSSL=T
AdminHost=localhost
Language=eng
Protocol=tcp
AllowSeparateNandCRules = T
NetworkFrame=   
SaveTime=   
DownTime=   
RuleTraceOn=   
RunningInBackground=T
GroupModeThreshold=0

**Any help would be appreciated**.

**Comment by rkvinoth** [comment]
Check if this helps: https://code.cubewise.com/tm1py-help-content/category/Configuration

**Comment by MarkoPuk01** [comment]
Yes, I also executed https://dw-tm1paw207.pi.local:1234/api/v1/$metadata successfully from that other machine.

**Comment by MarkoPuk01** [comment]
#import TM1py services
from TM1py.Services import TM1Service
from TM1py.Utils import Utils

#TM1 credentials
ADDRESS = "https://dw-tm1paw207.pi.local"
PORT = 1234
USER = "user"
PWD = "pass"
SSL = True

#Connect to the TM1 instance
tm1 = TM1Service(address=ADDRESS, port=PORT, user=USER, password=PWD, ssl=SSL)

**Comment by MarkoPuk01** [comment]
Thank you so much for your time, I didn't realize that could be problem.
